package.json

```diff
@@ -24,17 +24,6 @@
     "eslint": ">= 5"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.1",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-prettier": "^3.3.0",
-    "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-prettier": "^3.0.0",
-    "eslint-plugin-react": "^7.11.1",
-    "prettier": "^1.15.3"
-  },
-  "devDependencies": {
-    "eslint": "^5.10.0"
+    "eslint-plugin-jsdoc": "^8.0.1"
```

Comment:
Doesn't this need to be a primary dependency to be picked up by a dependent component?

Reply:
it is.